flow/scripts/floorplan.tcl

```diff
@@ -30,12 +30,6 @@ proc report_unused_masters {} {
 
 report_unused_masters
 
-#Run check_setup
-puts "\n=========================================================================="
-puts "Floorplan check_setup"
-puts "--------------------------------------------------------------------------"
-check_setup
```

Comment:
This appears to be unconditionally removed not just when skipping.

Reply:
Yes, but why is it done twice in the first place?

Removing the, I hope, superfluous and unconditional and leaving one conditional in report_metrics.tcl is the idea here.

Reply:
@oharboe normally users want to know early in the flow if there is a major sdc problem vs waiting to the end. If you don't check the logs until the end its not needed. If you monitor your log file and want to kill it on a major error then its needed. I think its good to make this an option.